script.js

```diff
@@ -1,4 +1,122 @@
+/*Base
+Requirements
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.*/
+
+/*
+Welcome Player 1.
+You rolled 3 for Dice 1 and 6 for Dice 2.
+Choose the order of the dice.
+
+Player 1, you chose Dice 2 first.
+Your number is 63.
+It is now Player 2's turn.
+*/
+
+//game mode
+var modeDiceRoll = "Start";
+var modeDiceOrder = "Userinput";
+var modeGameWinnner = "Winner";
+//default game mode
+var gameMode = modeDiceRoll;
+// players
+var playerOne = "Player One";
+var playerTwo = "Player Two";
+// arrays to store dice rolls
+var playerOneDice = [];
+var playerTwoDice = [];
+// player dice order
+var playerOneOrder;
+var playerTwoOrder;
+//win counter
+var playerOneWin = 0;
+var playerTwoWin = 0;
```

Comment:
Good use of globals. However just note that whenever we're building apps for multiple users we should prepare to have more than 2 user/players.  Might be best to use data types like arrays and objects that can hold multiple values early on when coding so it's not so much of a headache when refactoring :)